This is a doctest that checks the add_integer function
>>> add_integer = __import__('0-add_integer').add_integer

>>> add_integer(1, 2)
3

>>> add_integer(100, -2)
98

>>> add_integer(1)
99

>>> add_integer(100.3, -2)
98

>>> try:
...     add_integer(3, "School")
... except Exception as e:
...     print(e)
b must be an integer

>>> try:
...     add_integer(2, None)
... except Exception as e:
...     print(e)
b must be an integer

>>> try:
...    add_integer(None)
... except Exception as e:
...    print(e)
a must be an integer

>>> try:
...     add_integer(0)
... except Exception as e:
...     print(e)
98

>>> try:
...     add_integer('a')
... except Exception as e:
...     print(e)
a must be an integer

>>> try:
...     print(add_integer("School", 4))
... except Exception as e:
...     print(e)
a must be an integer

>>> try:
...     print(add_integer(None, 4))
... except Exception as e:
...     print(e)
a must be an integer

>>> try:
...     print(add_integer("School", "holberton"))
... except Exception as e:
...     print(e)
a must be an integer

>>> try:
...     print(add_integer([1, 2, 3, 4, 5], "holberton"))
... except Exception as e:
...     print(e)
a must be an integer

>>> try:
...     print(add_integer((1, 2), "holberton"))
... except Exception as e:
...     print(e)
a must be an integer

>>> try:
...     print(add_integer({1 : 2}, "holberton"))
... except Exception as e:
...     print(e)
a must be an integer
